src/vsr.zig

```diff
@@ -218,14 +215,29 @@ pub const Command = enum(u8) {
     request_blocks = 19,
     block = 20,
 
-    request_sync_checkpoint = 21,
-    sync_checkpoint = 22,
-
     start_view = 23,
 
+    // If a command is removed from the protocol, its ordinal is added here and can't be re-used.
+    const gaps = .{
+        12, // start_view without checkpoint
+        21, // request_sync_checkpoint
+        22, // sync_checkpoint
+    };
```

Comment:
The principle I am following here is that, ideally, our production code doesn't have "extras" --- there's no buggify, there's no compatability hacks (eventually), etc. 

So, for deprecated messages, I want to handle them exactly as the unknown messages, rather than, say, ignoring and logging.